org-formation/organization.yaml

```diff
@@ -399,6 +406,7 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         Department: IBC
         Project: Infrastructure
+        CostCenter: NO PROGRAM / 000000
```

Comment:
Since nextflow-tower supports multiple projects, see the granular budget report comment. Do we leave out the CostCenter tag on the account level if there is already more granular budget reporting?

Reply:
We can consider resource-level cost center tags to override the default, account level tag.  From this perspective I think it's fine to add `NO PROGRAM / 000000` as an account-level tag for the Nextflow account.